### DISCUSSION TITLE
Import Webradio AddOn generates error

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#53] [Fixed Issues]
Hello,

After updating to rAudio 1 20201230, I tried importing my webradios using the import Webradio addon.   The import process appeared to read all the entries in my pls file correctly, but generated the following error (according to the addon progress log):

**./importwebradio.sh: line 24: /srv/http/data/webradios/${url//\//|}: ambiguous redirect**

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by ceejayemm** [comment]
Jim

I have never used this feature but I might in future.  Do you have all your radio stations in one .pls file or one for each 'station' ?.  How do you name the file(s) ?

Thanks

Chris

**Reply by Sagnasty** [reply]
Chris,

I have all my Webradio stations in one pls file.

Details can be found by clicking on "source" from the Import Webradio Addon.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Reply by rern** [reply]
Webradio files to import must be in standard RuneAudio format:
```
[playlist]
NumberOfEntries=1
File1=http://urlpath:port
Title1=name
```
One file for each station.

**Reply by Sagnasty** [reply]
@rern 

Thanks for the clarification.  Could you update the README.md file as this currently indicates that single or multiple items per file is allowed.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by ceejayemm** [comment]
@Sagnasty /@rern

Thanks for this information. I have been able to upload my webradios using separate .pls files however with one file I got the same issue as Jim. It turned out that there was an additional space at the end of the url (on the line File1=). Once I removed the space the file uploaded correctly.

Chris

**Reply by Sagnasty** [reply]
@ceejayemm - good to hear you had success!

@rern 

It would be outstanding to have complementary Webradio Export / Import Addons - trying to maintain Webradios on 4 systems is getting painful ;-)

Also would be great to have the ability to export / import the Webradio coverart images and bindings as well.

I am okay to manually setup the Webradio on the first system (Name, URL, and cover art), but I no longer have a way to transfer this setup to my other systems.   In RuneAudio+R I used the backup / restore functionality  to transfer the Webradio setup between systems, but this no longer works in rAudio 1.

I also tried to copy the webradios and webradioimg directories between two 7-zip sessions, but the URLs stored as filenames is changed during the copy process (thanks windows!).   This is what eventually led me to the Webradio pls file import (to preserve the Webradio URL's in a pls file).

I am open to any suggestions / ideas that you might have.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+
